api/ServiceWorkerMessageEvent.json

```diff
@@ -0,0 +1,367 @@
+{
+  "api": {
+    "ServiceWorkerMessageEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerMessageEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "45"
+          },
+          "chrome_android": {
+            "version_added": "45"
+          },
+          "edge": {
+            "version_added": "17",
+            "notes": "Service workers is available in Microsoft Edge starting EdgeHTML 16 behind a flag."
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "44",
+            "version_removed": "55",
+            "notes": "Service workers (and Push) have been disabled in the Firefox 45 and 52 Extended Support Releases (ESR)."
```

Comment:
I would add a 2nd note saying that: "In Firefox 55 and later, the standard \<code>MessageEvent\</code> interface must be used instead." (only on the interface)

Reply:
Actioning the rest of this now